.github/workflows/validate-stacks.yaml

```diff
@@ -163,17 +163,17 @@ jobs:
         with:
           go-version: "1.19"
 
-      - name: Install odo v3
-        uses: redhat-actions/openshift-tools-installer@2de9a80cf012ad0601021515481d433b91ef8fd5 # v1
-        with:
-          odo: "3.15.0"
+      - name: Install odo latest version
+        run: |
+          curl -L https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-nightly-builds/odo-linux-amd64 -o odo
+          sudo install -m 0755 odo /usr/local/bin/odo
+          mkdir -p $HOME/bin 
+          cp ./odo $HOME/bin/odo
+          export PATH=$PATH:$HOME/bin
 
       - name: Install Ginkgo
         run: go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.13.0
 
-      - name: Check odo version
-        run: odo version
```

Comment:
Even though its nightly builds, might be still good to print the version in case we need to know the revision of the build during investigations.

Can see the nightly build revision `cc028de02-nightly` in the version printout I did locally:
```
./odo version
===============================================
⚠  unable to fetch the cluster server version
===============================================
odo v3.15.0 (cc028de02-nightly)

Podman Client: 4.9.4
```